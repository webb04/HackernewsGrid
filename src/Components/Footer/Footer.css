.Footer {
    padding: 20px;
    box-sizing: border-box;
    border-top: solid 2px rgb(255, 102, 0);
    background: #F6F6EF;
    height: 80px;
}

.Footer-title {
    margin: 0;
}