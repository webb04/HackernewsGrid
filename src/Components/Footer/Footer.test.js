import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import renderer from 'react-test-renderer';

it('Renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Renders correctly', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
});