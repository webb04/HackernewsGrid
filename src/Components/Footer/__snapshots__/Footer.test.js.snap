// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<footer
  className="Footer"
/>
`;

exports[`renders correctly 1`] = `
<footer
  className="Footer"
/>
`;
